packages/block-library/src/button/color-edit.js

```diff
@@ -193,7 +193,9 @@ function ColorEdit( props ) {
 
 	return (
 		<ColorPanel
-			enableContrastChecking={ isWebPlatform && ! gradient && ! style?.color?.gradient }
+			enableContrastChecking={
+				isWebPlatform && ! gradient && ! style?.color?.gradient
```

Comment:
This can be removed since that check is moved to the `ControlPanel`
```suggestion
				! gradient && ! style?.color?.gradient
```